ql/src/test/results/clientpositive/llap/limit_join_transpose.q.out

```diff
@@ -108,7 +111,7 @@ limit 1
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@src
 #### A masked pattern was here ####
-0	val_0	0	val_0
+238	val_238	238	val_238
```

Comment:
Does `-- SORT_QUERY_RESULTS` help avoid resultset change?

Reply:
I'll try but I don't think it will help in this case. My understanding is that SORT_QUERY_RESULTS sorts the rows on the standard output on the client/driver side.